bridge/engine/ActiveMediaList.cpp

```diff
@@ -94,6 +100,7 @@ bool ActiveMediaList::addAudioParticipant(const size_t endpointIdHash)
     uint32_t ssrc;
     if (!_audioSsrcs.pop(ssrc))
```

Comment:
Using the existing audioSsrcs, which are audio.lastN+2, 5 as default, should not be a problem.